services/dynamic/dynamic-xml.service.js

```diff
@@ -75,7 +75,7 @@ export default class DynamicXml extends BaseService {
     const pathIsAttr = (
       pathExpression.split('/').slice(-1)[0] || ''
     ).startsWith('@')
-    const parsed = new DOMParser().parseFromString(buffer)
+    const parsed = new DOMParser().parseFromString(buffer, 'text/xml')
```

Comment:
Notes on breaking change:
https://github.com/xmldom/xmldom/blob/master/CHANGELOG.md#fixed-27
In principle, this might be breaking if any users were using the dynamic xml badge to try and parse HTML documents (which may or may not have worked). One thing we could do here is try and decide whether we should pass `text/html` here based on the filename or response headers. I think my instinct on this is to just hard-code `text/xml` here, deploy it, and see if anyone pops up demanding we [re-enable spacebar heating](https://xkcd.com/1172/)

Reply:
Sounds reasonable to me.